### ISSUE TITLE
Split Page doesn't work for Japanese books

### ISSUE THREAD

**Issue by mrobe** [ScanTailor-Advanced/scantailor-advanced#62]
Japanese books order pages from "back to front", so the split page function mixes up all the page numbers.

There needs to be some kind of option to reverse the order.

**Comment by darnn** [comment]
When creating a new project, try ticking "Right to left layout".

**Comment by mrobe** [comment]
Thanks for that pointer. 

Is this documented somewhere?

**Comment by darnn** [comment]
I don't think so... I learned to use ST by poking around and just noticed it myself. Googling seems to turn this up, but of course, it's not the official documentation: https://digitalorientalist.com/2020/03/09/scantailor-installation-instructions-and-impressions/

**Comment by pablogventura** [comment]
We're tracking the **right-to-left / Japanese book** ordering pain for **Split page** and page numbering.

Before anyone implements a fix, it would help to lock the **desired UX**:

1. **Thumbnail order only** — reverse how pages appear in the strip while keeping internal processing as today?  
2. **Split logic only** — keep thumbnails as now but change how split assigns left/right or page indices?  
3. **Both**

Please reply with **1 / 2 / 3** (or a short alternative). Screenshots or a tiny sample project name pattern also help.

This issue stays open until we have that agreement and a concrete patch.

**Comment by pablogventura** [comment]
Implementation and **feedback request** are in **PR #133**: https://github.com/ScanTailor-Advanced/scantailor-advanced/pull/133

Please try a current build from that branch (or CI artifacts when available) and comment here or on the PR with **works / issues / regressions** for your use case.
